packages/bento-web/src/profile/ProfileEditPage/ProfileEditPage.tsx

```diff
@@ -1,52 +1,87 @@
-import { useMemo, useState } from 'react';
+import axios from 'axios';
+import { useCallback, useEffect, useMemo, useState } from 'react';
 import styled from 'styled-components';
 
+import { AssetRatioListItem } from '@/dashboard/components/AssetRatioListItem';
+
+import { useProfile } from '../ProfileDetailPage/hooks/useProfile';
 import { FieldInput } from '../components/FieldInput';
 import { FieldTextArea } from '../components/FieldTextArea';
 import { ExampleUserProfile } from '../constants/ExampleUserProfile';
+// import { profile } from '../constants/profile';
 import { ProfileLink } from '../types/UserProfile';
 import { Preview } from './components/Preview';
 import { ProfileLinkEditItem } from './components/ProfileLinkEditItem';
 import { TabBar } from './components/TabBar';
 
 const ManagePage = () => {
-  const [username, setUsername] = useState<string>(ExampleUserProfile.username);
-  const [displayName, setDisplayName] = useState<string>(
-    ExampleUserProfile.displayName,
+  const [profile, _] = useProfile();
+
+  const [username, setUsername] = useState<string>(profile?.username ?? '');
+  const [display_name, setDisplayName] = useState<string>(
+    profile?.display_name ?? '',
   );
-  const [bio, setBio] = useState<string>(ExampleUserProfile.bio);
-  const [links, setLinks] = useState<ProfileLink[]>(ExampleUserProfile.links);
+  const [bio, setBio] = useState<string>(profile?.bio ?? '');
+  const [links, setLinks] = useState<ProfileLink[]>(profile?.links ?? []);
+
+  console.log(profile, links);
+
+  useEffect(() => {
+    if (!profile) {
+      return;
+    }
+    setLinks(profile.links);
+  }, [profile]);
 
   const profileDraft = useMemo(
-    () => ({ ...ExampleUserProfile, username, displayName, bio, links }),
-    [username, displayName, bio, links],
+    () => ({
+      ...profile,
+      username,
+      display_name,
+      bio,
+      links,
+      images: ExampleUserProfile.images,
+      verified: false,
+      tabs: [],
+    }),
+    [username, display_name, bio, links],
   );
 
+  const onSubmit = useCallback(async () => {
+    const { data } = await axios.post(`/api/profile`, {
+      username,
+      display_name,
+      bio,
+      links,
+    });
+    console?.log(data);
+  }, [username, display_name, bio, links]);
+
   return (
     <>
       <Wrapper>
         <Preview profileDraft={profileDraft} />
         <EditorWrapper>
-          <TabBar />
+          <TabBar onClick={onSubmit} />
           <Container>
             <ProfileContainer id="profile">
               <FieldInput
                 field="사용자 이름"
                 placeholder="여러분의 링크에 들어가는 이름이에요"
-                defaultValue={ExampleUserProfile.username}
-                onChange={(e) => setUsername(e.target.value)}
+                defaultValue={profile?.username}
```

Comment:
```suggestion
                defaultValue={profile?.username ?? ''}
```
기본값이라는 의미니까 명시적으로 해봅시다! 아래도 적용